src/lua/assoc.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#pragma once
+
+#include <lua.h>
+
+#include "core/assoc.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif /* defined(__cplusplus) */
+
+/**
+ * Helper for looking up Lua specific strings in `mh_strnu32_t` hash table.
+ */
+static inline mh_int_t
+mh_strnu32_lua_find_str(struct mh_strnu32_t *h, const char *str, uint32_t len)
```

Comment:
This function and the entire file are now unused. Can be deleted.

Reply:
Dropped the unnecessary changes.